ci/ray_ci/automation/update_version.py

```diff
@@ -1,28 +1,25 @@
-import os
-
 import click
 
 from ci.ray_ci.automation.update_version_lib import (
     get_current_version,
     update_file_version,
 )
 
-bazel_workspace_dir = os.environ.get("BUILD_WORKSPACE_DIRECTORY", "")
-
 
 @click.command()
+@click.option("--root_dir", required=True, type=str)
```

Comment:
what should be the `root_dir` values provided by the users? is this script used somewhere and need to be updated?

Reply:
root_dir would be the path to Ray repository that user would want the changes to be on. For automation bot, it would clone Ray repo now and pass the path to the repo as root_dir here. If user ran the script locally, root_dir would be their Bazel workspace path.